src/config/options.ts

```diff
@@ -28,6 +28,7 @@ export type AppsignalOptions = {
   logLevel: string
   loggingEndpoint: string
   name: string
+  opentelemetryPort: string | number
```

Comment:
I was wondering why this was declared as `string | number`, because I thought `number` would make sense for this, but then I saw that it's also declared as `str | int` [in the Python integration](https://github.com/appsignal/appsignal-python/blob/main/src/appsignal/config.py#L51).

Then I kept looking. 🙃 A few lines below in this file, `statsdPort` is declared as `string` only. But the [stand-alone agent TOML configuration file](https://github.com/appsignal/appsignal-agent/blob/e6dc07f98a2de165e0bf5a40ecaba9b7044bc88f/agent/src/standalone_config.rs#L70-L79) expects a number, not a string. And [the Ruby configuration option docs](https://docs.appsignal.com/ruby/configuration/options.html#option-statsd_port) claims it should be a number.

I think we should aim to unify this across integrations -- it makes for a confusing experience for it to be typed differently in each one.

But for this PR, I think we should make `opentelemetryPort` and `statsdPort` have the same type, whether that's `string` or `number` or `string | number`.